lib/contentTypeParser.js

```diff
@@ -141,7 +141,7 @@ ContentTypeParser.prototype.run = function (contentType, handler, request, reply
   const resource = new AsyncResource('content-type-parser:run', request)
 
   if (parser === undefined) {
-    reply.send(new FST_ERR_CTP_INVALID_MEDIA_TYPE(contentType))
+    reply.send(new FST_ERR_CTP_INVALID_MEDIA_TYPE(contentType || undefined))
```

Comment:
Why does it need ` || undefined`?
If it is `contentType` is `undefined`, it will be `undefined || undefined`.

Reply:
I would change this line. The error should respect what it is, the message will be wrong if someone doing `reply.type('')`
https://github.com/xtx1130/fastify/blob/ff4ea7cc54bdf5053d812b03bf0d5374558cb34a/lib/handleRequest.js#L36

Reply:
> I would change this line. The error should respect what it is, the message will be wrong if someone doing `reply.type('')` https://github.com/xtx1130/fastify/blob/ff4ea7cc54bdf5053d812b03bf0d5374558cb34a/lib/handleRequest.js#L36

`contentType ` must be a string, so there use an empty string instead of  `undefined`